src/amber/validations/params.cr

```diff
@@ -122,4 +122,4 @@ module Amber::Validators
       @rules << rule
     end
   end
-end
+end
```

Comment:
Do we have a guideline on this? I like having a newline at the end of every file.